remake.yml

```diff
@@ -8,6 +8,7 @@ packages:
   - cowplot
   - scipiper
   - purrrlyr
+  - lubridate
```

Comment:
I'd keep this package declaration as close to the code that needs it as you can - it's used in tally_site_obs() only. This would be more important for a package that not everybody has installed and that not everybody should _need_ to have installed, e.g., a spatial library that's only needed within a task table that just one person will be running.